src/views/AskForHelp.jsx

```diff
@@ -24,6 +25,15 @@ export default function AskForHelp() {
 
   const isHotline = user && user.uid === hotlineId;
```

Comment:
```suggestion
  const isHotline = useMemo(() => user && user.uid === hotlineId, [user, hotlineId]);
```

If we include this constant in the dependency array below, we should memorize it, otherwise the useMemo hook below will be recomputed unnecessarily